commands.go

```diff
@@ -642,3 +643,22 @@ func fnCleanupLostPortals(ce *WrappedCommandEvent) {
 	}
 	ce.Reply("Finished cleaning up portals")
 }
+
+var cmdSyncPortal = &commands.FullHandler{
+	Func: wrapCommand(fnSyncPortal),
+	Name: "sync-portal",
+	Help: commands.HelpMeta{
+		Section:     HelpSectionPortalManagement,
+		Description: "Sync this portal.",
+	},
+	RequiresPortal: true,
+	RequiresLogin:  true,
+	RequiresAdmin:  true,
+}
+
+func fnSyncPortal(ce *WrappedCommandEvent) {
+	info, _ := ce.User.Client.RetrieveGroupByID(ce.Ctx, ce.Portal.GroupID(), ce.Portal.Revision)
+	if info != nil {
```

Comment:
This should log errors and reply if it fails or succeeds

Reply:
I didn't intend the sync-portal command for merge, so I didn't bother to make it pretty. It should probably be integrated in a more general `sync` command at some point, I just needed a way to trigger the sync of a single portal for testing.